extensions/wrapper/client/src/test/java/de/sovity/edc/client/DashboardPageApiServiceTest.java

```diff
@@ -0,0 +1,178 @@
+
+import java.util.List;
+
+import static de.sovity.edc.client.TestUtils.PROTOCOL_ENDPOINT;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.eclipse.edc.connector.contract.spi.types.negotiation.ContractNegotiation.Type.CONSUMER;
+import static org.eclipse.edc.connector.contract.spi.types.negotiation.ContractNegotiation.Type.PROVIDER;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@ApiTest
+@ExtendWith(EdcExtension.class)
+class DashboardPageApiServiceTest {
+    EdcClient client;
+    AssetIndex assetIndex;
+    PolicyDefinitionService policyDefinitionService;
+    TransferProcessService transferProcessService;
+    ContractNegotiationStore contractNegotiationStore;
+
+    @BeforeEach
+    void setUp(EdcExtension extension) {
+        assetIndex = mock(AssetIndex.class);
+        extension.registerServiceMock(AssetIndex.class, assetIndex);
+
+        policyDefinitionService = mock(PolicyDefinitionService.class);
+        extension.registerServiceMock(PolicyDefinitionService.class, policyDefinitionService);
+
+        transferProcessService = mock(TransferProcessService.class);
+        extension.registerServiceMock(TransferProcessService.class, transferProcessService);
+
+        contractNegotiationStore = mock(ContractNegotiationStore.class);
+        extension.registerServiceMock(ContractNegotiationStore.class, contractNegotiationStore);
+
+        TestUtils.setupExtension(extension);
+        client = TestUtils.edcClient();
+    }
+
+
+    @Test
+    void testDashboardPage() {
+
+        // arrange
+        var assets = List.of(
+                dummyAsset(),
+                dummyAsset(),
+                dummyAsset(),
+                dummyAsset(),
+                dummyAsset(),
+                dummyAsset()
+        );
+        var transferProcesses = List.of(
+                mockTransferProcess("test1", 1, TransferProcessStates.COMPLETED.code()),
+                mockTransferProcess("test2", 2, TransferProcessStates.TERMINATED.code()),
+                mockTransferProcess("test3", 3, TransferProcessStates.STARTED.code()),
+                mockTransferProcess("test4", 4, TransferProcessStates.DEPROVISIONED.code()),
+                mockTransferProcess("test5", 5, TransferProcessStates.TERMINATED.code()),
+                mockTransferProcess("test6", 6, TransferProcessStates.INITIAL.code())
+
+        );
+
+        var policyDefinitions = List.of(
+                mockPolicyDefinition(),
+                mockPolicyDefinition()
+        );
+
+        var contractNegotiations = List.of(
+                mockContractNegotiation(1, CONSUMER),
+                mockContractNegotiation(2, CONSUMER),
+                mockContractNegotiation(3, CONSUMER),
+                mockContractNegotiation(4, PROVIDER),
+                mockContractNegotiation(5, PROVIDER),
+                mockContractNegotiation(6, PROVIDER)
```

Comment:
remove one providing contract negotiation to create an imbalance, we want different numbers for stuff